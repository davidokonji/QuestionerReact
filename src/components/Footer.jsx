import React from 'react';
import '../style/footer.scss';

const Footer = () => {
  const date = new Date().getFullYear();
  return (
    <React.Fragment>
      <footer>
      &copy;
        {date}

      </footer>
    </React.Fragment>
  );
};

export default Footer;
