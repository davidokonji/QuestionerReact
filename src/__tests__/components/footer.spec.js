import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/footer.component';

describe('Footer component', () => {
  test('should render footer component correctly ', () => {
    const wrapper = shallow(<Footer />);

    expect(wrapper).toMatchSnapshot();
  });
});
