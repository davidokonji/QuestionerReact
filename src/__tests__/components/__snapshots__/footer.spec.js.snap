// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render footer component correctly  1`] = `
<footer
  className="py-4 footer page-footer fixed-bottom mt-5"
  id="sticky-footer"
>
  <div
    className="container text-center"
  >
    <small>
      Created with
      <img
        alt="love"
        src={
          Object {
            "canInstrument": true,
            "createTransformer": [Function],
            "getCacheKey": [Function],
            "process": [Function],
          }
        }
        style={
          Object {
            "marginLeft": "5px",
            "marginRight": "5px",
            "width": "12px",
          }
        }
      />
      by Okonji Chukwunonso
    </small>
  </div>
</footer>
`;
